project/src/CameraController.gd

```diff
@@ -13,4 +16,9 @@ func move(delta) -> void:
 	if Input.is_action_pressed("ui_left"):
 		position.x -= speed * delta
 	if Input.is_action_pressed("ui_right"):
-		position.x += speed * delta
\ No newline at end of file
+		position.x += speed * delta
+
+func signal_cam_pos_updated() -> void:
+	frame = ++frame
```

Comment:
++ doesn't pre-increment in gdscript. I learned there's not a post-increment either. As far as I can tell, we would have to use += or = and +.  Therefore, this number is always 0. I also can't figure out what ++ does and why it does not throw an error. Let me know if you figure this out.

Reply:
Omg that is wild.  I'll look into it.